src/js/tables.js

```diff
@@ -1,11 +1,14 @@
-/*global require*/
 /**
  * Wrap tables
  */
-'use strict';
 
-var $ = require('jquery');
+'use strict';
 
-$(function() {
-	$('.o-techdocs-content > table').wrap('<div class="o-techdocs-table-wrapper"></div>');
-});
+document.addEventListener('o.DOMContentLoaded', function() {
+	Array.from(document.querySelectorAll('.o-techdocs-content > table')).forEach(function(el) {
+		var wrapper = document.createElement('div');
+		wrapper.className = "o-techdocs-table-wrapper";
```

Comment:
I don't really care too much about it in JS. JS is easy to make readable searchable with a mix of `, simple quotes and double quotes.

In Sass it is annoying me: it compiles to double quotes, but I prefer single quotes because they are easier to type. Also, in CSS the codebase needs to be easily searchable (since there is no logic for IDEs to parse and reference) so only having one type of quotes helps.